config/src/main/kotlin/org/springframework/security/config/web/servlet/HttpSecurityDsl.kt

```diff
@@ -44,12 +44,16 @@ import javax.servlet.http.HttpServletRequest
  *          formLogin {
  *              loginPage = "/log-in"
  *          }
+ *          securityContext {
```

Comment:
We shouldn't include setting the `securityContext` in this example, because it is meant to be a very basic example.